hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ClusterStateEndpoint.java

```diff
@@ -134,15 +150,19 @@ public Response getClusterState() {
       totalKeys += fileRecord.getValue();
     }
     builder.setKeys(totalKeys);
-
+    // Subtract deleted containers from total containers.
+    containerStateCounts.setTotalContainerCount(
```

Comment:
@ArafatKhan2198 , pls call this setter method for total container count once at last for final value. You can remove the call from line no 95 above.

Reply:
Thanks for the suggestion @devmadhuu will implement it !!